RetailCoder.VBE/Inspections/QuickFixes/EncapsulateFieldQuickFix.cs

```diff
@@ -32,7 +32,7 @@ public override void Fix()
         {
             var vbe = _target.Project.VBE;
 
-            using (var view = new EncapsulateFieldDialog(_state, _indenter))
+            using (var view = new EncapsulateFieldDialog(new EncapsulateFieldViewModel(_state, _indenter)))
```

Comment:
I'm thinking we should inject everything, up to the inspection's constructor - and eventually give ourselves some `IInspectionResultFactory<TInspectionResult>`; when we do, we won't have to be bothered with UI couplings, and the dependencies will move from the inspection to the factory's constructor... and if we manage to wire up a Ninject auto-factory, we won't even need to think about them.